ecl/hql/hqlmeta.cpp

```diff
@@ -1540,13 +1540,13 @@ static IHqlExpression * createSubSorted(IHqlExpression * dataset, IHqlExpression
     if (!isLocal && !alwaysLocal)
         subsort.setown(convertSubSortToGroupedSort(subsort));
 
-    assertex(isAlreadySorted(subsort, order, isLocal||alwaysLocal, ignoreGrouping));
+    assertex(isAlreadySorted(subsort, order, isLocal||alwaysLocal, ignoreGrouping, false));
     return subsort.getClear();
 }
 
 IHqlExpression * getSubSort(IHqlExpression * dataset, const HqlExprArray & order, bool isLocal, bool ignoreGrouping, bool alwaysLocal)
 {
-    if (isAlreadySorted(dataset, order, isLocal||alwaysLocal, ignoreGrouping))
+    if (isAlreadySorted(dataset, order, isLocal||alwaysLocal, ignoreGrouping, true))  // could possible have requireDistribution = false
```

Comment:
possible -> possibly
Would it be a good idea to expand on this comment and state why?